### ISSUE TITLE
`gulp deploy-lambda` no longer works since nodejs4.3 is no longer supported

### ISSUE THREAD

**Issue by forevermatt** [shevchenkos/DynamoDbBackUp#65]
Attempting to run `gulp deploy-lambda ...`, I received the following error message:

```
[12:49:16] 'deploy-lambda' errored after 6.05 s
[12:49:16] InvalidParameterValueException: The runtime parameter of nodejs4.3 is no longer supported for creating or updating AWS Lambda functions. We recommend you use the new runtime (nodejs10.x) while creating or updating functions.
```

Since AWS now rejects a Runtime value of "nodejs4.3" (and "nodejs6.10"), the `lambda.js` file needs to be updated (along with the `lambda/package-lock.json` dependencies, presumably) for a later version of NodeJS.

I plan to submit a PR for this, but wanted to start with an Issue in case any discussion needs to happen first, or other aspects of the problem I haven't anticipated are brought to light.

**Comment by thoean** [comment]
We didn't invest time in this library anymore due to support for point in time recovery natively by AWS dynamoDb. See https://docs.aws.amazon.com/amazondynamodb/latest/developerguide/PointInTimeRecovery_Howitworks.html

If you still believe there's value fixing this, I'm happy to accept a PR (even though we'll need @shevchenkos to publish it to npm).

**Comment by forevermatt** [comment]
Thanks for the quick response. Since the Point-In-Time recovery options for DynamoDB restore to a new table, I found that this DynamoDbBackUp library was an easier way to restore data... reconfiguring my Serverless application to use a different name for it's DynamoDB tables was going to be hard/problematic.

I'll still submit a PR for this, as I'm planning on continuing to use (a fork of) this library for the foreseeable future.

**Comment by forevermatt** [comment]
Fixed in #66
